### ISSUE TITLE
How to use bottleneck layer? 

### ISSUE THREAD

**Issue by huangjun0413** [mravanelli/pytorch-kaldi#95]
How to use bottleneck layer like this LinearTransform:
`<AffineTransform> <InputDim> 129 <OutputDim> 1024 <BiasMean> -2.000000 <BiasRange> 4.000000 <ParamStddev> 0.042080 <MaxNorm> 0.000000         
<Sigmoid> <InputDim> 1024 <OutputDim> 1024      
<AffineTransform> <InputDim> 1024 <OutputDim> 1024 <BiasMean> -2.000000 <BiasRange> 4.000000 <ParamStddev> 0.109375 <MaxNorm> 0.000000      
<Sigmoid> <InputDim> 1024 <OutputDim> 1024     
<AffineTransform> <InputDim> 1024 <OutputDim> 1024 <BiasMean> -2.000000 <BiasRange> 4.000000 <ParamStddev> 0.109375 <MaxNorm> 0.000000     
<Sigmoid> <InputDim> 1024 <OutputDim> 1024    
<AffineTransform> <InputDim> 1024 <OutputDim> 1024 <BiasMean> -2.000000 <BiasRange> 4.000000 <ParamStddev> 0.109375 <MaxNorm> 0.000000   
<Sigmoid> <InputDim> 1024 <OutputDim> 1024    
<LinearTransform> <InputDim> 1024 <OutputDim> 50 <ParamStddev> 0.113277 <LearnRateCoef> 0.100000    
<AffineTransform> <InputDim> 50 <OutputDim> 1024 <BiasMean> -2.000000 <BiasRange> 4.000000 <ParamStddev> 0.113277 <LearnRateCoef> 0.100000 <BiasLearnRateCoef> 0.100000 <MaxNorm> 0.000000          
<Sigmoid> <InputDim> 1024 <OutputDim> 1024      
<AffineTransform> <InputDim> 1024 <OutputDim> 653 <BiasMean> 0.000000 <BiasRange> 0.000000 <ParamStddev> 0.120869 <LearnRateCoef> 1.000000 <BiasLearnRateCoef> 0.100000       
<Softmax> <InputDim> 653 <OutputDim> 653`

**Comment by huangjun0413** [comment]
set dnn_act = linear
